source/fundamentals/authentication.txt

```diff
@@ -1,119 +1,124 @@
-.. uses authentication.rst
+.. _pymongo-auth:
 
-Authentication Examples
-=======================
+Authentication Mechanisms
+=========================
 
-MongoDB supports several different authentication mechanisms. These examples
-cover all authentication methods currently supported by PyMongo, documenting
-Python module and MongoDB version dependencies.
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
 
-.. _percent escaped:
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: authorize, boto
 
-Percent-Escaping Username and Password
---------------------------------------
+MongoDB supports several different authentication mechanisms. This guide
+discusses all authentication methods currently supported by PyMongo.
```

Comment:
```suggestion
discusses all authentication methods currently supported by {+driver-short+}.
```

Reply:
i've started just leaving this as is, with plans to change on the second pass, since they don't affect the final page